### ISSUE TITLE
Open link

### ISSUE THREAD

**Issue by davidbenucci** [wavebox/waveboxapp#101]
* Which version of Wavebox are you using? [e.g. 3.0.0]
3.1.5

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Ubuntu 17.04

When I click on a link, it simply opens the browser at start page and stop, without going to the destination url. I am using chrome as browser. With  firefox the problem disappears.

**Comment by Thomas101** [comment]
Hi, thanks for reporting. I've tried this my end and it seems to work. Do you know which version of Chrome you have installed?

**Comment by davidbenucci** [comment]
Chrome 58.0.3029.81 unknown (64-bit) under Ubuntu 17.04

**Comment by Thomas101** [comment]
Thanks, I'll see if I can reproduce it

**Comment by Thomas101** [comment]
@davidbenucci are you still seeing this issue? Some users have reporting that not having `xdg-utils` results in this behaviour

**Comment by simiansim** [comment]
I am experiencing this as well.  58.0.3029.110 (64-bit) Ubuntu 16.04 Linux 4.4.0-78-generic
xdg-utils (xdg-open) is installed. 

I actually have 2 versions installed
3.1.5 installed from wavebox.io (working)
3.1.8 installed via "snap" package (broken) 

Doing the following will fix the issue. (But forcing this extra step on the user is also a problem.) 
sudo apt install snapd-xdg-open

Maybe you should follow these instructions:
https://www.freedesktop.org/wiki/Software/xdg-utils/
"Although we expect that these scripts will generally come as part of the operating system, we recommend that you package the scripts that your application needs along with your product as a fallback."

Basically I think the issue is a snap based app can not use a non-span based browser...
http://www.ktechpit.com/linux/using-xdg-open-snap-package-open-external-url/
http://www.omgubuntu.co.uk/2017/05/corebird-ubuntu-snap-app-available

**Comment by Thomas101** [comment]
Hi, as regards snap packages `snapd-xdg-open` needs to be installed alongside Wavebox and other electron based apps. There are some open issues upstream in the snap project to address this, but until then we're sticking with this.

Going to close this off for now as there's a solution to open links with the snap version. Thanks!
